functions/go/set-namespace-v2/main.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"os"
+	"strings"
+
+	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
+)
+
+func main() {
+	if err := fn.AsMain(fn.ResourceListProcessorFunc(run)); err != nil {
+		os.Exit(1)
+	}
+}
+
+func run(rl *fn.ResourceList) error {
```

Comment:
Same comment as in https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/768#discussion_r826346186
`main` module is not importable.

Reply:
oh, right! Good catch, thank you!